ngraph/python/tests/test_ngraph/test_basic.py

```diff
@@ -271,3 +271,89 @@ def test_result():
 
     result = run_op_node([node], ng.ops.result)
     assert np.allclose(result, node)
+
+
+def test_node_output():
+    input_array = np.array([0, 1, 2, 3, 4, 5])
+    splits = 3
+    expected_shape = len(input_array) // splits
+
+    input_tensor = ng.constant(input_array, dtype=np.int32)
+    axis = ng.constant(0, dtype=np.int64)
+    split_node = ng.split(input_tensor, axis, splits)
+
+    split_node_outputs = split_node.outputs()
+
+    assert len(split_node_outputs) == split_node.get_output_size()
```

Comment:
I think I would rather compare with the `splits` variable here. You get both values from the `split_node` here and if for some reason it splits the input tensor to 2 instead of 3, this assert will not catch it.

Reply:
That's a valid point, changed with the newest commit.